pages/entropy/create-your-first-entropy-app.mdx

```diff
@@ -240,53 +251,48 @@ async main() {
 
 The code snippet above generates a random number and hashes it. The resulting hash is the user commitment. The code calls the Entropy contract to get the fee required for requesting a random number. Then it calls the request method of the CoinFlip contract with the `userCommitment` as an argument and the required fee. Finally, the code reads the sequenceNumber from the `FlipRequested` event emitted by the CoinFlip contract.
 
-Next, add a snippet to get the provider’s random number.
-
-```javascript copy
-async main() {
-  ...
-  // Fetch the provider uri
-  const info = await entropyContract.methods
-    .getProviderInfo(process.env["PROVIDER_ADDRESS"])
-    .call();
-  const providerUri = web3.utils.hexToString(info.uri);
-
-  // Fetch provider commitment
-  const res = await fetch(`${providerUri}/revelations/${sequenceNumber}`);
-  const fortunaRevelation = await res.json();
-}
-```
-
-The code first fetches the provider service URI from the Entropy contract and then fetches the provider’s random number from the service.
-
 Finally, add the following code snippet to get the flip result.
 
 ```javascript copy
 async main() {
   ...
-  // Fulfill the request
-  const fulfillReceipt = await coinFlipContract.methods
-    .fulfill(sequenceNumber, randomNumber, "0x" + fortunaRevelation.value.data)
-    .send({
-      from: address,
+  let fromBlock = requestReceipt.blockNumber;
```

Comment:
I think all the `...` items should be code comments to the effect of `// ... prior code omitted` for 100% clarity